File: Program/1_Array/Array Standard problem/easy/8_count_the_number_of_possible_triangles.py
def count_tringle(arr):
    size = len(arr)
    count = 0
    for i in range(size):
        for j in range(i+1,size):
            for k in range(j+1, size):
                if arr[i] + arr[j] > arr[k] and arr[j] + arr[k] > arr[i] and arr[k] + arr[i] > arr[j]:
                    print(arr[i], arr[j], arr[k])
                    count+=1
    return count


l = [5,7,3,6,10,4]
print("Total Number of tringle:",count_tringle(l))

File: Program/1_Array/Array Standard problem/easy/8_count_the_number_of_possible_triangles.c
#include <stdio.h>
#include<stdlib.h>
int find_trinagle(int arr[], int size)
{
    int count = 0;
    printf("All tringle pair :\n");
    for(int i=0; i<size; i++)
        {
            for(int j=i+1; j<size; j++)
                {
                    for(int k=j+1; k<size; k++)
                        {
                            if(arr[i]+arr[j] > arr[k] && 
                                arr[i]+arr[k] > arr[j] &&
                                arr[k] + arr[j] > arr[i]
                            )
                            {
                                printf("%d %d %d",arr[i], arr[j], arr[k]);
                                printf("\n");
                                count++;
                            }
                        }
                }
        }
    printf("\nTotal number of possible tringle :%d", count);

}

void main()
{
    int arr[] = {5,7,3,6,10,4};
    int size = sizeof(arr)/sizeof(int);
    find_trinagle(arr, size);
 
}